dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -105,11 +105,10 @@ protected void startCommon(AlluxioConfiguration conf,
     try (FileSystem fs = createBaseFileSystem(fsContext, fuseOptions)) {
       AlluxioJniFuseFileSystem fuseFileSystem = createFuseFileSystem(fsContext, fs, fuseOptions);
       setupFuseFileSystem(fuseFileSystem);
-      launchFuse(fuseFileSystem, fsContext, fuseOptions, true); // This will block until umount
+      // This will block on serving until fuse gets destroyed
+      launchFuse(fuseFileSystem, fsContext, fuseOptions, true);
     } catch (Throwable t) {
```

Comment:
Out of scope of this pr, but we should not swallow fatal error (such as jvm oom error)

Reply:
yeah this is actually catching throwable at starting fuse with fuse_main_real, jvm oom throws from some other threads, but will get the exit(-1) back